src/Okta.Sdk/Generated/IUserActivationToken.Generated.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace Okta.Sdk
 {
+    /// <summary>Contains the method contract for UserActivationToken resources.</summary>
```

Comment:
The class is `UserActivationToken`, the interface is `IUserActivationToken`.

I can update this phrase to be more clear.